build_test_local.sh

```diff
@@ -7,11 +7,14 @@ if [[ $(uname -m) == 'arm64' ]];
 then
   echo "Apple Silicon detected"
   export DOCKER_BUILDKIT=1
+  export LOCAL_PLATFORM=linux/amd64
```

Comment:
is this something you spoke about with @jsugarman ? I wondered how it fitted in with his comments about this line not working for him?

Reply:
had a session with @jsugarman and we found this works as intended.